From 22404ba0d9b002685c47271b164a22dbe8dc7856 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Fri, 9 Mar 2012 21:16:51 +0800
Subject: [PATCH 474/678] ENGR00176347-6 mx6solo sabreauto: remove build
 warning

Remove the following build warning:
mx6q_sabreauto.c: In function 'enet_board_init':
mx6q_sabreauto.c:999: warning: unused variable 'reg'
mx6q_sabreauto.c: At top level:
mx6q_sabreauto.c:921: warning: 'phy_read' defined but not used

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 board/freescale/mx6q_sabreauto/mx6q_sabreauto.c |    7 +++++--
 1 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/board/freescale/mx6q_sabreauto/mx6q_sabreauto.c b/board/freescale/mx6q_sabreauto/mx6q_sabreauto.c
index f221564..33f232a 100644
--- a/board/freescale/mx6q_sabreauto/mx6q_sabreauto.c
+++ b/board/freescale/mx6q_sabreauto/mx6q_sabreauto.c
@@ -907,6 +907,10 @@ int board_late_init(void)
 	return 0;
 }
 
+/*
+ * The following function is used for debug purpose
+ */
+/*
 static int phy_read(char *devname, unsigned char addr, unsigned char reg,
 		    unsigned short *pdata)
 {
@@ -916,6 +920,7 @@ static int phy_read(char *devname, unsigned char addr, unsigned char reg,
 		       devname, addr, reg);
 	return ret;
 }
+*/
 
 static int phy_write(char *devname, unsigned char addr, unsigned char reg,
 		     unsigned short value)
@@ -985,8 +990,6 @@ iomux_v3_cfg_t enet_pads[] = {
 
 void enet_board_init(void)
 {
-	unsigned int reg;
-
 	mxc_iomux_v3_setup_multiple_pads(enet_pads,
 			ARRAY_SIZE(enet_pads));
 	set_gpio_output_val(GPIO6_BASE_ADDR, (1 << 24), 1);
-- 
1.7.7.4

